lab-ron-dunphy/index.js

```diff
@@ -0,0 +1,7 @@
+'use strict';
+
+const program = require('./lib/read-files.js');
+
+module.exports = exports = {};
+
+program();
```

Comment:
There's no anonymous function callback in this function call, nor are you logging the output. At this point it's not going to be functional.